resources/definitions/diy220.def.json

```diff
@@ -0,0 +1,89 @@
+{
+    "version": 2,
+    "name": "Diytech 220",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Venkatkamesh",
+        "manufacturer": "Sri Vignan Technologies",
+        "weight": 3,
+        "file_formats": "text/x-gcode",
+        "platform": "ultimaker3_platform.obj",
+        "platform_texture": "svtbacktext.png",
+        "platform_offset": [0, 0, 0],
+        "has_materials": true,
+        "has_variants": true,
+        "preferred_variant_name": "0.4 mm",
+        "machine_extruder_trains":
+        {
+            "0": "diy220_extruder_0"
+        }
+    },
+    "overrides": {
+        "machine_name": { "default_value": "Diytech 220" },
+        "machine_start_gcode" : {
+            "value": "\"\"  if machine_gcode_flavor == \"UltiGCode\" else \"G21 ;metric values\\nG90 ;absolute positioning\\nM82 ;set extruder to absolute mode\\nM107 ;start with the fan off\\nG28 Z0 ;move Z to bottom endstops\\nG28 X0 Y0 ;move X/Y to endstops\\nG1 X15 Y0 F4000 ;move X/Y to front of printer\\nG1 Z15.0 F9000 ;move the platform to 15mm\\nG92 E0 ;zero the extruded length\\nG1 F200 E50 ;extrude 10 mm of feed stock\\nG92 E0 ;zero the extruded length again\\nG1 F9000\\n;Put printing message on LCD screen\\nM117 Printing...\""
```

Comment:
Does the machine support UltiGcode? If not, why have two different start gcodes?

Reply:
Using ultimaker board 
In title itself we defined as did(do it yourself)  model
Functions are same as ultimaker

Reply:
Huh? Using Ultimaker board? Is this an Ultimaker clone? 

I've also not been able to find the printer available anywhere. Is it even a printer that is sold somewhere?

Reply:
We have sold diy models to educational institutions around 60qty
Between sorry it's mks board using marlin architecture 
Other are industrial models sold for industries around our state.